ADQL.tex

```diff
@@ -1866,7 +1796,7 @@ \subsubsection{CONTAINS}
 is in, or on, the boundary of the second argument and the numeric value 0
 if it is not.
 
-When used as a predicate in the WHERE clause of a query, the numeric return
+When used as a predicate in the WHERE clause of a query, the numeric returned
```

Comment:
I think this needs to be "the numeric value (or better: integer) returned"

Reply:
Done